agent/Cargo.toml

```diff
@@ -17,9 +17,9 @@ tracing-subscriber = { version = "0.3", features = ["registry", "env-filter"] }
 tracing-opentelemetry = "0.16"
 
 # k8s-openapi must match the version required by kube and enable a k8s version feature
-k8s-openapi = { version = "0.14.0", default-features = false, features = ["v1_20"] }
+k8s-openapi = { version = "0.16.0", default-features = false, features = ["v1_20"] }
 # "rustls-tls" feature must be used kube 0.71.0 to build correct k8s client for IPv6 cluster
```

Comment:
Need to reconcile this comment

Reply:
We're safe to remove this comment. Kube 0.75.0 won't break IPV6 cluster.